server/index.ts

```diff
@@ -56,18 +75,39 @@ export const configSchema = schema.object({
     keepalive: schema.boolean({ defaultValue: true }),
   }),
   auth: schema.object({
-    type: schema.string({
-      defaultValue: '',
-      validate(value) {
-        if (
-          !['', 'basicauth', 'jwt', 'openid', 'saml', 'proxy', 'kerberos', 'proxycache'].includes(
-            value
-          )
-        ) {
-          return `allowed auth.type are ['', 'basicauth', 'jwt', 'openid', 'saml', 'proxy', 'kerberos', 'proxycache']`;
-        }
-      },
-    }),
+    type: schema.oneOf(
+      [
+        schema.arrayOf(schema.string(), {
+          defaultValue: [''],
+          validate(value: string[]) {
+            if (!value || value.length === 0) {
+              return `Authentication type is not configurred properly.`;
```

Comment:
configurred -> configured

Reply:
I would also add a more specific hint such as `Authentication type is not configured properly. At least one authentication method must be selected.`

Reply:
Addressed.

Reply:
same question, do you mean you have had the change, but have not pushed yet?

Reply:
Sorry for the confusion. Mark it as resolved for the tracking purpose from my side. Will push this changes in the next commit.